src/main/resources/db/migration/V3_0__fix_melding_type_innkommende_legeerklaring.sql

```diff
@@ -0,0 +1,30 @@
+UPDATE melding SET type='FORESPORSEL_PASIENT_LEGEERKLARING' WHERE
+innkommende AND type='FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER'
+AND uuid IN (
+'a93aba64-06b4-4b60-8fb1-054f87407457',
```

Comment:
Har hentet ut uuid'er for meldingene som har kommet på de samme conversationRef'ene som Lars fant fram.